src/components/ConfigMap.js

```diff
@@ -1,14 +1,63 @@
+import {receipts_data} from "../data/receipts_data";
+import {expenditure_data} from "../data/expenditure_data";
+
 let appConfig = {
 	"app.title" : "Story Generator",
 	"app.version" : "ALPHA"
-	
+		
 };
 
 let homeComponent = {
-	"primary_header": "OpenBudgets India",
-	"secondary_header" : "Story Generator",
+	"primary_header": "OpenBudgets India", // Text Only
+	"secondary_header" : "Story Generator", // Text 
 	"description" : "Select an indicator to generate the visualizations"
 }
 
+let leftSideBarComponent = {
+	"top_logo": {
+		"format" : "text", // text/img
+		"text_config" : {
+			"text" : "Story Generator", // Null for img and text value if top-logo-format is  
+			"release_version" : "ALPHA" // Alpha/PreALPHA/BETA	
+		},
+		"img_config" :{
+			"source": "https://raw.githubusercontent.com/cbgaindia/portal-design/master/logo_OBI/logo_types/light_bg_logo/draft_final/logo_with_text/draft_final.png", // null if text else provide link for images.	
+			"width": "270",
+			"height" : "",
+		}
+	},
+	"bottom_logo" : { // To represent organization logo.
+		"img_config" :{
+			"source": "https://raw.githubusercontent.com/cbgaindia/portal-design/master/logo_OBI/logo_types/light_bg_logo/draft_final/logo_with_text/draft_final.png", // null if text else provide link for images.	
+			"width": "250", //Image width
+			"height" : "", // Image height
+			"link" : "https://openbudgetsindia.org/" // Hyperlink to the organization
+		}
+	},
+	"selection_panel" :{
+		"panels" : [
+		{
+			"isActive" : "true",
```

Comment:
Its a tradeoff. This would provide the flexibility of taking in data from other heterogeneous sources. We would aim to reduce these parameters over the period as we evolve.